### ISSUE TITLE
WALC v0.1.2 does not start on Arch Based Distros

### ISSUE THREAD

**Issue by cryptosteve2** [cstayyab/WALC#10]
Hi,
downloaded walc-AppImage, made it executable und tried to run:

> [stell @ sorum:~/bin/walc]% ./WALC.AppImage   
[13400:0428/085735.345774:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /tmp/.mount_WALC.A682nG7/chrome-sandbox is owned by root and has mode 4755.
zsh: trace trap (core dumped)  ./WALC.AppImage

I cannot see any problems with /tmp
drwxrwxrwt 26 root root 620 28. Apr 08:57 /tmp

Btw: thanks for WALC :)

**Comment by cstayyab** [comment]
have you tried the trivial method of restarting the app?

**Comment by cstayyab** [comment]
Try an older version if the error still exist then it is due to the chromium/chrome installed on your PC.

**Comment by cryptosteve2** [comment]
sorry, what's the "trivial method"? 
the older version (WALC.0.1.1.AppImage) works. 
I'm running WALC on Manjaro 18.0.4 with Gnome 3.32.

**Comment by cstayyab** [comment]
by "trivial method" I just meant to restart the app.

Anyways, try running it with  non-root user. I tried to use on a "root" user and it is not starting. I just want to confirm the issue so I can report it to relevant repository.

**Comment by cryptosteve2** [comment]
I always run AppImages as a non root user. stell is my non-privileged user.

> [**stell** @ sorum:~/bin/walc]% ./WALC.AppImage

**Comment by cstayyab** [comment]
remove `/tmp/.mount_WALC.*` directory using
```
sudo rm -R /tmp/.mount_WALC*
```
then re-run WALC

**Comment by cryptosteve2** [comment]
Sorry, forgot to say that ... there is no directory in /tmp containing .mount_WLAC* - the only /tmp/.mount* is from another AppImage (not related to WALC).

**Comment by cstayyab** [comment]
Please try to clone repo and run it using npm start:

Here is what you have to do:

```
git clone https://github.com/cstayyab/WALC.git
cd WALC
npm install
npm start
```

if errror occurs please post the output here.

**Comment by cryptosteve2** [comment]
Done that but that seems to be the same error

> [stell@sorum WALC]$ npm start
>
> WALC@0.1.2 start /home/stell/WALC 
> electron .  
>
> [2543:0428/133247.138100:FATAL:setuid_sandbox_host.cc(157)] The SUID sandbox helper binary was found, but is not configured correctly. Rather than run without sandboxing I'm aborting now. You need to make sure that /home/stell/WALC/node_modules/electron/dist/chrome-sandbox is owned by root and has mode 4755.

**Comment by cstayyab** [comment]
Do you have Google Chrome or Chromium already installed on your system? If yes, Please tell me the version of it.

**Comment by cryptosteve2** [comment]
google-chrome 74.0.3729.108-1is installed, chromium isn't.

**Comment by cstayyab** [comment]
Okay, Finally I have confirmed the issue on my Manjaro 18.0.2
I guess this is an issue with electron-builder (which I am using to build AppImage) or Google Chrome.

**Comment by cstayyab** [comment]
### TL;DR
**Fixed. Download the beta release [here](https://github.com/cstayyab/WALC/releases/tag/v0.1.3-beta)** Please confirm the fix so I can close this issue.
### Details
The error was in the latest version (5.0.0) of electron which they reverted back to 4.1.5. I have also reverted it back. Now it is perfectly running on my Manjaro.

**Comment by cryptosteve2** [comment]
The beta fixed it for me. Thanks a lot.
